### ISSUE TITLE
`prettierPath` resolved against the wrong directory

### ISSUE THREAD

**Issue by frantisekhanzlikbl** [prettier/prettier-vscode#2590]
# What have I tried

1. overwrite the VSCode setting `prettier.prettierPath` in a VSCode multi-root workspace (my concrete use-case is yarn PnP with the yarn prettier SDK, i.e.: `"prettier.prettierPath": ".yarn/sdks/prettier/index.js"`)
1. attempt to format a file in any of the workspace members. 

# What did I expect

The file gets formatted with my preferred prettier version

# What has happened instead

prettier-vscode spits out the error `Cannot find module '{path to my workpsace root}/subpackage/.yarn/sdks/prettier/index.js'`.

The path is obviously wrong, as the path should be relative to the workspace root, regardless of what file I have opened.

# Notes

- I'd be willing to implement a fix if the team doesn't have time to resolve this, but I have no experience with VSCode extension development or APIs, so I would probably need a moderate amount of guidance.
- I am not sure if this is within the extension developer's hands, and it's just a minor inconvenience for me, but with a multi-root workspace open, setting `prettierPath` in `.vscode/settings.json` doesn't work and must be set in the workspace file instead.
- As a work-around, creating a symlink pointing to the right path under each workspace root works (at least on Linux)

**Comment by ntotten** [comment]
This doesn't seem like the right config for yarn pnp. Have you tried using this config: https://yarnpkg.com/getting-started/editor-sdks

**Comment by frantisekhanzlikbl** [comment]
Sorry for not getting back to you earlier, life got in the way as usual. :smile: 

Unfortunately, I believe I have followed the Yarn instructions precisely, and ended up with exactly that config.

I am not too interested in this any more, though, as VSCode's multi-root workspaces have a buttload of other problems, so I eventually ended up using normal config again. I'll leave it up to you to whether to keep this open or not, as I am unsure if this is truly a bug on the extension's side or if I have misconfigured something. On the other hand, if more testing would be helpful, I'd be happy to help as much as I can. :slightly_smiling_face:
